### ISSUE TITLE
2.19.0 - TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

### ISSUE THREAD

**Issue by akalex** [requests/requests#4684]
When running python program with command line arguments **-OO** the exception occurs. That issue can be reproduced since **requests** version **2.19.0**, since the logic inside module **requests/status_codes.py** has changed.

## Expected Result
python3.6 -OO -m my_script.py runs without any exceptions

## Actual Result

**TypeError** exception raises
```
Traceback (most recent call last):
  File "/Users/alexandromyshev/Downloads/send_requests.py", line 1, in <module>
    import requests
  File "/Users/alexandromyshev/venvs/rtest/lib/python3.6/site-packages/requests/__init__.py", line 114, in <module>
    from .models import Request, Response, PreparedRequest
  File "/Users/alexandromyshev/venvs/rtest/lib/python3.6/site-packages/requests/models.py", line 43, in <module>
    from .status_codes import codes
  File "/Users/alexandromyshev/venvs/rtest/lib/python3.6/site-packages/requests/status_codes.py", line 119, in <module>
    _init()
  File "/Users/alexandromyshev/venvs/rtest/lib/python3.6/site-packages/requests/status_codes.py", line 116, in _init
    __doc__ = (__doc__ + '\n' +
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```
## Reproduction Steps

1) Create some simple script like
```import requests

def main():
    response = requests.get('http://google.com')
    print(response.status_code)
    return response


if __name__ == '__main__':
    main()
```
2) Run the command
```
python3.6 -OO send_requests.py
```

## System Information

    $ python3.6 -m requests.help

```
{
  "chardet": {
    "version": "3.0.4"
  },
  "cryptography": {
    "version": ""
  },
  "idna": {
    "version": "2.7"
  },
  "implementation": {
    "name": "CPython",
    "version": "3.6.4"
  },
  "platform": {
    "release": "17.5.0",
    "system": "Darwin"
  },
  "pyOpenSSL": {
    "openssl_version": "",
    "version": null
  },
  "requests": {
    "version": "2.19.0"
  },
  "system_ssl": {
    "version": "100020df"
  },
  "urllib3": {
    "version": "1.23"
  },
  "using_pyopenssl": false
}

```

**Comment by sigmavirus24** [comment]
Fixed in #4683

**Comment by nateprewitt** [comment]
2.19.1 is pushed to pypi, so this should be resolved now. Thanks everyone!
